Source/DafnyRuntime/DafnyRuntimeJava/src/main/java/dafny/DafnySequence.java

```diff
@@ -4,7 +4,6 @@
 import java.util.*;
```

Comment:
Should we be checking in this jar file?  It's not obvious that it corresponds to these changes.

Reply:
I've been checking it in.  I seem to recall something failing in CI when I didn't once.  I'm not sure why it's in version control to begin with.

Reply:
OK, if it's breaking things for you let's update it, but it would be ideal to remove it from git sometime soon.